src/matchers/toHaveBeenCalledOnceWith.js

```diff
@@ -16,7 +16,9 @@ export function toHaveBeenCalledOnceWith(received, expected) {
   }
 
   const passOnce = received.mock.calls.length === 1;
-  const pass = passOnce && this.equals(expected, received.mock.calls[0][0]);
+  const mockCall = received.mock.calls[0];
+  const receivedArray = Array.isArray(mockCall) ? mockCall : [mockCall];
```

Comment:
It's always an array.

Reply:
Ah. I could have sworn I saw a case that wasn't, but I don't know what I thought I saw now 😛 
Removed the ternary.